web/impact/impact/tests/test_cancel_office_hour_reservation.py

```diff
@@ -119,7 +116,19 @@ def test_user_attempts_to_cancel_non_existent_reservation(self):
         office_hour = MentorProgramOfficeHourFactory(finalist=None)
         response = self._submit_cancellation(office_hour,
                                              user=self.make_user())
-        notification = {'detail': DEFAULT_PERMISSION_DENIED_DETAIL}
+        notification = {
+            'detail': NO_SUCH_RESERVATION,
+            'header': FAIL_HEADER,
+            'success': False}
+        self.assertEqual(response.data, notification)
```

Comment:
can we use the existing `self.assert_ui_notification(response, expected_success, expected detail)` format?